src/domain/entities/LandingNode.ts

```diff
@@ -16,6 +16,7 @@ export interface LandingNode {
     type: LandingNodeType;
     icon: string;
     iconLocation: string;
+    pageRendering: string;
```

Comment:
Isn't limited to some values? (single, multiple), then we need a union. Similar to property `type`.